From 1d0ca0d869f66e3b27343e5df4c33585bb9384e6 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Mon, 1 Mar 2010 16:32:15 +1000
Patch-mainline: 2.6.34
References: fate#310916
Git-commit: 0de1a57bedce0899c98da4c0910d4f38c9b7c8f4
Subject: [PATCH 0476/2588] drm/radeon/kms: bump the KMS version number for
 square tiling support.

Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 0de1a57bedce0899c98da4c0910d4f38c9b7c8f4)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/radeon_drv.c |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/radeon/radeon_drv.c b/drivers/gpu/drm/radeon/radeon_drv.c
index f6456e0..6eec0ec 100644
--- a/drivers/gpu/drm/radeon/radeon_drv.c
+++ b/drivers/gpu/drm/radeon/radeon_drv.c
@@ -40,9 +40,11 @@
 
 /*
  * KMS wrapper.
+ * - 2.0.0 - initial interface
+ * - 2.1.0 - add square tiling interface
  */
 #define KMS_DRIVER_MAJOR	2
-#define KMS_DRIVER_MINOR	0
+#define KMS_DRIVER_MINOR	1
 #define KMS_DRIVER_PATCHLEVEL	0
 int radeon_driver_load_kms(struct drm_device *dev, unsigned long flags);
 int radeon_driver_unload_kms(struct drm_device *dev);
-- 
1.7.6

